.github/CODEOWNERS

```diff
@@ -122,7 +122,11 @@
 /docs/coding-guidelines/typescript* @Automattic/type-review
 
 # FSE (Full Site Editing)
-/apps/full-site-editing @Automattic/serenity @Automattic/cylon @Automattic/ajax
+/apps/full-site-editing/* @Automattic/serenity @Automattic/cylon @Automattic/ajax
+/apps/full-site-editing/full-site-editing-plugin/* @Automattic/serenity @Automattic/cylon @Automattic/ajax
+/apps/full-site-editing/full-site-editing-plugin/full-site-editing @Automattic/serenity @Automattic/cylon
+/apps/full-site-editing/full-site-editing-plugin/posts-list-block @Automattic/ajax
+/apps/full-site-editing/full-site-editing-plugin/starter-page-templates @Automattic/ajax
```

Comment:
Just noting here that we are being moved off of this project.

Do we need to "own" this @obenland?

Reply:
Until another team works on it, I'm afraid so

Reply:
@getdave https://www.youtube.com/watch?v=7qnd-hdmgfk